### DISCUSSION TITLE
使用cmder作为终端输出颜色由问题

### DISCUSSION THREAD

**Discussion by CKCat** [xmake-io/xmake#2974] [Q&A]
### Xmake 版本

2.7.2

### 操作系统版本和架构

Window 10

### 描述问题

在cmder 中执行 xmake 后，输出内容如下：
 $ xmake
checking for Microsoft Visual Studio (x64) version ... 2019
?[0m?[1;38;5;3;1mnote: ?[0minstall or modify (m) these packages (pass -y to skip confirm)??[0m
in xmake-repo:

### 期待的结果

希望输出结果与cmd中执行结果一致。

**Comment by waruqi** [comment]
xmake g --theme=plain
